src/main/java/lotto/controller/LottoController.java

```diff
@@ -0,0 +1,90 @@
+package lotto.controller;
+
+import lotto.dao.exception.DataAccessException;
+import lotto.service.LottoService;
+import lotto.service.PaymentInfoService;
+import lotto.service.dto.PaymentInfoDto;
+import spark.Request;
+import spark.Response;
+
+import java.sql.SQLDataException;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.IntStream;
+
+import static java.util.stream.Collectors.toList;
+import static lotto.controller.common.CommonController.nullable;
+import static lotto.controller.common.CommonController.render;
+
+public class LottoController {
+    private static final PaymentInfoService PAYMENT_INFO_SERVICE = PaymentInfoService.getInstance();
+    private static final LottoService LOTTO_SERVICE = LottoService.getInstance();
+
+    private LottoController() {
+        throw new AssertionError();
+    }
+
+    public static String goSelectLotto(Request request, Response response) throws SQLDataException {
```

Comment:
사용하지 않는 코드는 제거해 주세요.

```suggestion
    public static String goSelectLotto(Request request, Response response) {
```